src/encoded/tests/test_audit_replicate.py

```diff
@@ -65,8 +65,8 @@ def test_audit_inconsistent_modification_tag(
         library_1):
     testapp.patch_json(experiment['@id'], {'assay_term_name': 'ChIP-seq',
                                            'target': target_H3K27ac['@id']})
-    testapp.patch_json(target['@id'], {'investigated_as': ['recombinant protein'],
-                                       'label': 'FLAG'})
+    testapp.put_json(target['@id'], {'investigated_as': ['synthetic tag'],
```

Comment:
Do you really mean to `put` here?

Reply:
Yes. It needs to get rid of the `target_organism` property in target.